web/themes/joinup/templates/join-collection-form.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * Template to render the share content form.
```

Comment:
> could be named better, as it's not for joining, but seems mainly concerned about leaving

This form is created in /web/modules/custom/collection/src/Form/JoinCollectionForm.php and there are two functionalities - join and leave, there is name "JoinCollection" so I used it also for template.

> not placed in correct folder
There is a bug in the drupal core - https://www.drupal.org/node/2817103. The template is not detected if I place it in subfolder.